directory.json

```diff
@@ -42,6 +42,7 @@
 	"HLUG": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/China/Hangzhou(%E6%9D%AD%E5%B7%9E)LinuxUserGroup-api.json",
 	"HSG": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Singapore/HackerspaceSG",
 	"IDR": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Indonesia/IDRuby-api.json",
+	"IT": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Sri%20Lanka/MillenniumIT-api.json",
```

Comment:
@jig08 They are not. They are a software company. See my comments: https://github.com/fossasia/fossasia-communities/pull/101#issuecomment-173863801